### PR TITLE
Extract use accordion hook

### PR DESCRIPTION
**Pull request by JuliaHempel**
Extract thee useAccordion hook so that functionality can be used by other components as well.

### CODE REVIEW

File: src/components/organisms/Accordion/Accordion/Accordion.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ### Summary
 
-`Accordion` is just a small wrapper component around the `useAccordion` hook. It holds the accordion and passes it down through context to other [Accordion](#/Organisms/Accordion) components
+`Accordion` is just a small wrapper component around the `useExpader` hook. It holds the accordion and passes it down through context to other [Accordion](#/Organisms/Accordion) components
```
Comment (katebeavis) [inline src/components/organisms/Accordion/Accordion/Accordion.md:3]: Typo on expander

File: src/hooks/useExpander/useExpander.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+### Summary
+
+`useExpander()` can be used to expand and show hidden content while being accessible.
+
+It can be useful for accordions and in tables with expanding rows. To see it being used along with context see the [`<Accordion>`](/#/Components/Organisms/Accordion).
+
+### Examples
+
+```js
+import { Accordion, Text, useExpander } from '@zopauk/react-components';
```
Comment (katebeavis) [inline src/hooks/useExpander/useExpander.md:10]: Remove `Accordion`

**Comment by katebeavis** [comment]
I'm getting a compile error `Warning: src/hooks/useExpander/useExpander.ts matches a pattern defined in “components” or “sections” options in your style guide config but doesn’t export a component`. Also I can see that in the readme that useExapnder has an uppercase letter at the start `UseExpander` which I think is to do with the above error but I can't see where it is written like that 🤔

**Comment by JuliaHempel** [comment]
> I'm getting a compile error `Warning: src/hooks/useExpander/useExpander.ts matches a pattern defined in “components” or “sections” options in your style guide config but doesn’t export a component`. Also I can see that in the readme that useExpander has an uppercase letter at the start `UseExpander` which I think is to do with the above error but I can't see where it is written like that 🤔

I saw that every once in a while too but not on every compile. No idea how to fix it, as far as I know I do the same thing as useViewport.

**Review by endymion1818 — APPROVED** [review]

**Review by steedems — DISMISSED** [review]

File: src/hooks/useAccordion/useAccordion.ts [hunk 3]
```diff
@@ -38,7 +38,8 @@ export type IsActiveAccordionSection = (index: number) => boolean;
 
 type ActiveSections = number[];
 
-export const useAccordion = () => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const useAccordion = (props?: {}) => {
```
Comment (poteirard) [inline src/hooks/useAccordion/useAccordion.ts:42]: Sorted in #625

**Review by poteirard — APPROVED** [review]

**Pull request merged**